tests/components/energyzero/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,43 @@
+# serializer version: 1
```

Comment:
The fact that these files are auto-generated and can be updated with a single command is awesome.